src/caja-places-sidebar.c

```diff
@@ -2884,23 +2884,38 @@ bookmarks_button_release_event_cb (GtkWidget *widget,
     }
 
     tree_view = GTK_TREE_VIEW (widget);
-    model = gtk_tree_view_get_model (tree_view);
 
     if (event->button == 1)
     {
-
         if (event->window != gtk_tree_view_get_bin_window (tree_view))
         {
             return FALSE;
         }
 
+        model = gtk_tree_view_get_model (tree_view);
```

Comment:
I got the warning: `space before tab in indent` in this line. Could you fix this please?

Reply:
This should already be fixed by the third commit (079102fcb47fad3d34ad98be9136863d0400b166)

Reply:
Seems OK.